src/core/notifyManager.ts

```diff
@@ -94,3 +94,6 @@ class NotifyManager {
 // SINGLETON
 
 export const notifyManager = new NotifyManager()
+
+// For testing purposes (not change made by jest.setup.js)
+export const notifyManagerTest = new NotifyManager()
```

Comment:
can we tie this to `process.env.NODE_ENV !== 'production'`, or maybe even better, create the `new NotifyManager()` inside the test?

Reply:
Done in https://github.com/tannerlinsley/react-query/pull/2876/commits/d3e40aff30fdb8d967c5e5c793e5b4b7757c6630
I had to export `NotifyManager` class. 
I think it's the best solution for testing.